content/exchange/artifacts/Windows.EventLogs.Hayabusa.yaml

```diff
@@ -105,9 +114,12 @@ sources:
         WHERE log(message=Stdout)
 
         -- Upload the raw file.
-        SELECT upload(file=CSVFile) AS Uploads FROM scope()
+        SELECT upload(file=ResultFile) AS Uploads FROM scope()
 
  - name: Results
    query: |
+        LET CSV_RESULT  <= SELECT * FROM parse_csv(filename=ResultFile)
```

Comment:
It is probably not ideal to parse these files into memory - use non matertialized LET instead